modules/apps/bookmarks/bookmarks-web/src/main/java/com/liferay/bookmarks/web/internal/info/item/renderer/BookmarksEntryFullContentInfoItemRenderer.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.bookmarks.web.internal.info.item.renderer;
+
+import com.liferay.bookmarks.constants.BookmarksWebKeys;
+import com.liferay.bookmarks.model.BookmarksEntry;
+import com.liferay.info.item.renderer.InfoItemRenderer;
+import com.liferay.petra.reflect.ReflectionUtil;
+import com.liferay.portal.kernel.language.LanguageUtil;
+
+import java.util.Locale;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Yang Cao
+ */
+@Component(enabled = false, service = InfoItemRenderer.class)
+public class BookmarksEntryFullContentInfoItemRenderer
+	implements InfoItemRenderer<BookmarksEntry> {
+
+	@Override
+	public String getLabel(Locale locale) {
+		return LanguageUtil.get(locale, "title");
+	}
+
+	@Override
+	public void render(
+		BookmarksEntry entry, HttpServletRequest httpServletRequest,
+		HttpServletResponse httpServletResponse) {
+
+		try {
+			httpServletRequest.setAttribute(
+				BookmarksWebKeys.BOOKMARKS_ENTRY, entry);
+
+			RequestDispatcher requestDispatcher =
+				_servletContext.getRequestDispatcher(
+					"/bookmarks/info/item/renderer/full_content.jsp");
+
+			requestDispatcher.include(httpServletRequest, httpServletResponse);
+		}
+		catch (Exception exception) {
+			ReflectionUtil.throwException(exception);
+		}
+	}
+
+	@Reference(
+		target = "(osgi.web.symbolicname=com.liferay.bookmarks.web)",
+		unbind = "-"
+	)
+	public void setServletContext(ServletContext servletContext) {
+		_servletContext = servletContext;
+	}
```

Comment:
Can you make this method protected, or alternatively, annotate the field? I don't see this public method used elsewhere.